routes/web.php

```diff
@@ -11,10 +11,24 @@
 |
 */
 
+use Illuminate\Routing\Router;
+
 Route::get('/', function () {
     return view('welcome');
 });
 
 Auth::routes();
 
 Route::get('/home', 'HomeController@index')->name('home');
+
+$router->group([
```

Comment:
Containing the api listing in the web.php as opposed to the api.php because it's quicker than setting up Passport to do OAuth and setting up creating api tokens etc with Laravel/Passport